client-runtime/src/main/java/com/microsoft/rest/v2/annotations/HostParam.java

```diff
@@ -52,5 +52,5 @@
      * encoded hence engine should not encode it, by default value will be encoded.
      * @return Whether or not this argument is already encoded.
      */
-    boolean encoded() default false;
+    boolean encoded() default true;
```

Comment:
Why change to true?

Reply:
Because if this was set to false, then if my interface had a Host annotation like `@Host("{baseUrl}")` and a function with a HostParam annotation like `@HostParam("baseUrl")` where the HostParam's value was "https://my.base.url", then the "://" would get encoded. That was breaking some of our internal logic. I think it's more likely that HostParam values will be passed in already encoded.